telegohandler/bot_handler.go

```diff
@@ -55,6 +50,11 @@ func NewBotHandler(bot *telego.Bot, updates <-chan telego.Update, options ...Bot
 // Start starts handling of updates
```

Comment:
Let's specify that calling `Start()` multiple times after the first one does nothing

Reply:
> 

get it！